### PR TITLE
[Ready] Makes Adv surgery tools more common around the station!

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs]
Adds in a cargo crate that has a tool belt form Med-co that has all the new surgery tools for 5000 credits
Adds a surgery belt that holds all the Adv surgery tools and some drapes
Adds the new surgery belt to CMO locker
Adds a new Surgery duffle bag for evil synda! This bag has the Adv surgery tools and a synda medical kit. Costs 15 CT //MITE Cost 18 later on if the meta is that
[why]
God damn it I want feed back on this stuff before we all forget about and its meta wise op as shit or something!
Nice to see some Qol things around as well, 'adds lore'-ish?

### CODE REVIEW

File: code/game/objects/items/storage/backpack.dm [hunk 1]
```diff
@@ -441,6 +441,23 @@
 	new /obj/item/clothing/mask/muzzle(src)
 	new /obj/item/mmi/syndie(src)
 
+/obj/item/storage/backpack/duffelbag/syndie/surgery_adv
```
Comment (Ragolution) [inline code/game/objects/items/storage/backpack.dm:444]: Doesn't make sense if these are 'researched' by the station.

File: code/game/objects/items/storage/backpack.dm [hunk 2]
```diff
@@ -441,6 +441,23 @@
 	new /obj/item/clothing/mask/muzzle(src)
 	new /obj/item/mmi/syndie(src)
 
+/obj/item/storage/backpack/duffelbag/syndie/surgery_adv
+	name = "advanced surgery duffel bag"
+	desc = "A large duffel bag, this one is for holding Med-Co latest surgery tools."
```
Comment (Ragolution) [inline code/game/objects/items/storage/backpack.dm]:
```suggestion
	desc = "A large duffel bag for holding surgical tools. Bears the logo of an advanced med-tech firm."
```

File: code/game/objects/items/storage/belt.dm [hunk 3]
```diff
@@ -168,6 +168,20 @@
 		/obj/item/pinpointer/crew
 		))
 
+
+/obj/item/storage/belt/medical/surgery_belt_adv
+	name = "Surgery belt"
```
Comment (Ragolution) [inline code/game/objects/items/storage/belt.dm]:
```suggestion
	name = "surgical supply belt"
```

File: code/game/objects/items/storage/belt.dm [hunk 4]
```diff
@@ -168,6 +168,20 @@
 		/obj/item/pinpointer/crew
 		))
 
+
+/obj/item/storage/belt/medical/surgery_belt_adv
+	name = "Surgery belt"
+	desc = "A normal medical belt that has been preloaded with advance surgery tools. Their is a tag on it that reminds people to place back the tools in there original pockets."
```
Comment (Ragolution) [inline code/game/objects/items/storage/belt.dm]:
```suggestion
	desc = "A specialized belt designed for holding surgical equipment. It seems to have specific pockets for each and every surgical tool you can think of."
```

Normal medical belts cannot hold surgical drills or circular saws so I hope you fixed that.

File: code/modules/cargo/packs.dm [hunk 5]
```diff
@@ -1153,6 +1153,24 @@
 					/obj/item/storage/firstaid/regular)
 	crate_name = "first aid kit crate"
 
+/datum/supply_pack/medical/iv_drip
+	name = "IV Drip Crate"
+	desc = "Contains a single IV drip for administering blood to patients."
```
Comment (Ragolution) [inline code/modules/cargo/packs.dm]:
```suggestion
	desc = "Contains a single IV drip stand for intravenous delivery."
```

Comment (Ragolution) [reply code/modules/cargo/packs.dm]: More descriptive of contents.

File: code/modules/cargo/packs.dm [hunk 6]
```diff
@@ -1153,6 +1153,24 @@
...
+	cost = 700
+	contains = list(/obj/machinery/iv_drip)
+	crate_name = "iv drip crate"
+
+/datum/supply_pack/science/adv_surgery_tools
```
Comment (Ragolution) [inline code/modules/cargo/packs.dm:1163]: Why even have it be a research node if they can just be ordered via cargo?

No.

File: code/modules/uplink/uplink_items.dm [hunk 7]
```diff
@@ -982,6 +982,13 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/storage/backpack/duffelbag/syndie/surgery
 	cost = 3
 
+/datum/uplink_item/device_tools/surgerybag_adv
+	name = "Syndicate Surgery Duffel Bag"
+	desc = "The Syndicate surgery duffel bag is a toolkit containing all newest surgery tools, surgical drapes, \
+			a Syndicate brand MMI, a straitjacket, a muzzle, and a full Syndicate Combat Medic Kit."
```
Comment (Ragolution) [inline code/modules/uplink/uplink_items.dm:988]: Unnecessary line break?

File: code/modules/uplink/uplink_items.dm [hunk 8]
```diff
@@ -982,6 +982,13 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
+	item = /obj/item/storage/backpack/duffelbag/syndie/surgery_adv
+	cost = 15 //Mite be to cheap
```
Comment (Ragolution) [inline code/modules/uplink/uplink_items.dm:990]:
```suggestion
	cost = 7
```

The base one costs 3. Do you think they're really 'good enough' to rationalize a 5x price cost?

**Review by Ragolution — CHANGES_REQUESTED** [review]

File: code/modules/uplink/uplink_items.dm (hunk 7 above)
Comment (Trilbyspaceclone) [reply code/modules/uplink/uplink_items.dm:988]: Same line brake as normal medical surgery kit

File: code/modules/uplink/uplink_items.dm (hunk 8 above)
Comment (Trilbyspaceclone) [reply code/modules/uplink/uplink_items.dm:990]: Yes there *that good*

File: code/modules/cargo/packs.dm (hunk 6 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm:1163]: You do know the point of me added that is so ***more*** people use the items

File: code/modules/cargo/packs.dm (hunk 5 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm]: I moved to to be A-Z like the commit at the top says

File: code/game/objects/items/storage/backpack.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/game/objects/items/storage/backpack.dm:444]: There not tho

File: code/game/objects/items/storage/belt.dm (hunk 4 above)
Comment (Trilbyspaceclone) [reply code/game/objects/items/storage/belt.dm]: What are you talking about they can ya dork

**Review by silicons — APPROVED** [review]

**Pull request merged**